src/app/login/page.js

```diff
@@ -1,13 +1,13 @@
 import Link from "next/link";
 
```

Comment:
If someone is on this page how is he/she going to navigate to sign up page in case they have changed their ming to login

Reply:
Reasonable concern but I wanted to update the ui for that on Figma before implementing it, so it will be in a separate PR. It'll be something like this

<img width="217" alt="image" src="https://github.com/content-is-queen/creator-platform/assets/14906802/52d34f3a-b17c-415c-8223-8f90e7a371ed">